here_for_change/municipalities/templates/municipalities/ward_detail.html

```diff
@@ -616,11 +616,47 @@ <h2 class="text-2xl md:text-3xl benfritz text-white text-center">
 ></script>
 <script>
```

Comment:
Data is fine to inline into the page but all other JavaScript should be in a separate file and bound properly.